### ISSUE TITLE
rip out brainjs entirely

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/ppComplete#54]
it's sad to see it go, because it really formed the impetus for the project, and was the first thing we worked on. 

but it's not maintained, and requires me to support a large separate codebase to train classifiers with it. 

using scikit-neuralnetwork is much more direct. it integrates nicely (theoretically) with sklearn, so we can use gridsearch cv. we can keep the same data formatting. it's actively maintained and improved. and it offers many more nn types we can train. 

in short, the choice is obvious. 
http://scikit-neuralnetwork.readthedocs.org/en/latest/guide_advanced.html#grid-search

**Comment by ClimbsRocks** [comment]
there's also this nn library wrapped to be compatible with scikit-learn:
https://groups.google.com/forum/#!topic/lasagne-users/mdFF3SL7-JQ
https://github.com/dnouri/nolearn

**Comment by ClimbsRocks** [comment]
finished, sadly. 

though these other nns train much more slowly, so i'm somewhat tempted to add brain.js back in :heart_eyes:
